### DISCUSSION TITLE
New industrial tiles

### DISCUSSION THREAD

**Discussion by cnub66** [SuperTux/supertux#2102] [Show and tell]
Moving from #1921 per request
I have redrawn the game's pre-existing industrial tileset to be less crunchy-looking and hopefully more fitting. I also drew some unisolid tiles that fit the industrial theme.
Thoughts, feedback, etc. welcome
![145237717-52d0ccb7-edf7-4d2f-958f-abe0530354f1](https://user-images.githubusercontent.com/95162186/149662891-8132f92b-05ff-4de6-8d4f-64736eef9b7b.jpg)

**Comment by VincentAraneo** [comment]
Cool Tiles!
